### PR TITLE
Feature/filter

### PR DESCRIPTION
**Pull request by Joepolymath**
Implemented the filter array method.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -35,4 +35,5 @@ This is to familiarize you with open source contribution, CI and of course Testi
 
 ## Contributors
 
-- [marvinjude](https://github.com.marvinjude) (map)
+- [marvinjude](https://github.com/marvinjude) (map)
+- [Joepolymath](https://github.com/Joepolymath) (map)
```
Comment (marvinjude) [inline README.md:39]:
```suggestion
- [Joepolymath](https://github.com/Joepolymath) (filter)
```

File: src/filter.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
+/**
+ *
+ * @param {funtion} callbackFn
```
Comment (marvinjude) [inline src/filter.js:3]:
```suggestion
 * @param {function} callbackFn
```

**Comment by marvinjude** [comment]
Nice one @Joepolymath! 👯 

I just fixed a couple of typos in the last commit 👍 

Now I'm seeing that the [contributors section](https://github.com/marvinjude/array-methods-playground#contributors) does not list other implemented methods. Can you push a commit for that?

**Comment by Joepolymath** [comment]
> Nice one @Joepolymath! 👯
> 
> I just fixed a couple of typos in the last commit 👍
> 
> Now I'm seeing that the [contributors section](https://github.com/marvinjude/array-methods-playground#contributors) does not list other implemented methods. Can you push a commit for that?

Sure. I am on it.

**Comment by Joepolymath** [comment]
> Yup! that works

Updated

**Comment by marvinjude** [comment]
Now that I'm looking at this project again, there are a lot of improvements one could possibly make 😄 e.g adding

- Linting
- Opinionated code formatting
- Test runner (Like Github actions, was using Travis CI before now but they no longer have a free plan hehe)

But I don't really think it's worth the effort at least on my end 😄 

Since the project is just for learning sake, I'm happy to review and merge PRs if it's something you ever wanna jump on 😉

**Comment by Joepolymath** [comment]
> Thanks for contributing!
> 
> Now that I'm looking at this project again, there are a lot of improvements one could possibly make 😄 e.g adding
> 
> * Linting
> * Opinionated code formatting
> * Test runner (Like Github actions, was using Travis CI before now but they no longer have a free plan hehe)
> 
> But I don't really think it's worth the effort at least on my end 😄
> 
> Since the project is just for learning sake, I'm happy to review and merge PRs if it's something you ever wanna jump on 😉

I understand. 
I could pick it up.

**Comment by marvinjude** [comment]
Go for it 🚀

**Comment by marvinjude** [comment]
Added a workflow for testing 👍

**Pull request merged**